### ISSUE TITLE
How to add DDP with val loader?

### ISSUE THREAD

**Issue by cswwp** [yangkky/distributed_tutorial#6]
How to add DDP with val evaluation? Is it same with train? @yangkky

**Comment by yangkky** [comment]
You'd have to find a way to compile the validation statistics onto one
process.

**Comment by rancheng** [comment]
This code is pretty clear:
https://github.com/rwightman/pytorch-image-models/blob/master/train.py#L730-L732

                reduced_loss = reduce_tensor(loss.data, args.world_size)
                acc1 = reduce_tensor(acc1, args.world_size)
                acc5 = reduce_tensor(acc5, args.world_size)
